dev_env/bootstrap.ps1

```
@@ -42,16 +42,23 @@ function Ensure-VirtualEnv() {
-  if ($existing -contains $VenvName) {
-    Write-Host "[ok] Virtualenv '$VenvName' já existe"
-  } else {
-    Write-Host "[info] Criando virtualenv '$VenvName'"
-    $list = (& pyenv install -l) -split "`n" | ForEach-Object { $_.Trim() }
-    $ver = $list | Where-Object { $_ -match '^3\.13\.[0-9]+$' } | Select-Object -Last 1
-    if (-not $ver) { throw "Não foi possível descobrir versão 3.13.x no pyenv-win." }
-    pyenv install -s $ver
-    pyenv virtualenv $ver $VenvName
-  }
+  if ($existing -contains $VenvName) {
+    Write-Host "[ok] Virtualenv '$VenvName' já existe"
```

PSScriptAnalyzer (1.24.0)

[warning] File 'bootstrap.ps1' uses Write-Host. Avoid using Write-Host because it might not work in all hosts, does not work when there is no host, and (prior to PS 5.0) cannot be suppressed, captured, or redirected. Instead, use Write-Output, Write-Verbose, or Write-Information.

(PSAvoidUsingWriteHost)